contentmgr/pinning.go

```diff
@@ -400,10 +400,31 @@ func (cm *ContentManager) handlePinningComplete(ctx context.Context, handle stri
 
 	if cont.Aggregate {
 		// this is used by staging content aggregate
+		if len(pincomp.Objects) != 1 {
```

Comment:
Aggregate are supposed to have more than 1 objects right? 

This will also pass if `len` = 0. Use `> 1` instead.

Reply:
aggregates should onnly ever 1 one object